build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+python src/etc/build_rust.py $@
```

Comment:
`from __future__ import print_function` (and other __future__ imports) is a good way to support both 2.7 and 3.*.